CHANGES_NEXT_RELEASE

```diff
@@ -2,4 +2,4 @@
 - Fix: wrong accumulation in counter sentOK
 - Fix: more descriptive error messages for CPr update/query forward fail
 - Add: reset for notification counters in threadpool mode
-
+- Add: alarm manager to keep a state about errors (Issue #1582)
```

Comment:
The alarm manager is an internal artifact. I'd express this in functional terms, maybe something like this:

"Add: proper alarm management, including activation/deactivation in logs (Issue #1582)"